### DISCUSSION TITLE
Would like to add a binary of Iris to my github repo.

### DISCUSSION THREAD

**Discussion by ghost** [IrisShaders/Iris#246] [General]

**Comment by coderbot16** [comment]
Hello,

You're definitely allowed to do so under the license (as long as you follow the LGPLv3, which includes providing a source link), however I do ask that you refrain from doing that at this time.

Essentially, I'm just not yet prepared to deal with all of the support requests and bug reports that would come from making Iris available to the majority of used, which is why I've not made compiled builds publicly available.

**In summary: you are legally allowed to, but I ask that you refrain from doing so until I've decided to release public builds myself.**
